src/E-Shop.Repositories/BaseRepository.cs

```diff
@@ -0,0 +1,54 @@
+﻿using E_Shop.Models;
+using E_Shop.Models.Shared;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace E_Shop.Repositories
```

Comment:
rename the namespace root name to `EShop`

Reply:
Ok